z_infra_provisioning/cloud_infra_center/ocp_upi/roles/configure-pre-check/tasks/main.yaml

```diff
@@ -87,4 +87,28 @@
 - name: Check server group quota
   fail:
     msg: "Server group number is not enough, current: {{ server_group.stdout_lines[0] }}, required: 1"
-  failed_when: "server_group.stdout_lines[0]|int < 1"
\ No newline at end of file
+  failed_when: "server_group.stdout_lines[0]|int < 1"
+
+- name: Get storage provider
+  shell: openstack volume service list -f csv | grep cinder-volume |grep -E 'enabled|up' |head -n1|awk -F , '{print$4$5}' |tr -d '"'
```

Comment:
this cmd is not checking the storage provider of the specified volume_type_id.
as long as there is one cinder-volume service up and running, the check will always be true. ()

suggest:
1. get the volume_backend_name from  "capabilities:volume_backend_name='v7k60'" in output of `openstack volume type show $$volume_type_id`
2. check this specific cinder-volume host status from output of 'openstack volume service list'

Reply:
is this done at line 111-114? 
@dongyanyang

Reply:
capabilities:volume_backend_name='v7k60' was file `cinder-<storagehostname>.conf` content, I can not get this file through ansible host, not any ICIC nodes.

I think this task was used to check the cinder-volume service, I will remove the status check for storage provider.

And for the volume_type_id we have another task to check it.
```
- name: Get storage template
  shell: openstack volume type list -c ID -f value | grep {{ volume_type_id }} | wc -l
  register: storage_template
  when:  
  - volume_type_id is defined
  - disk_type == "scsi"
```